Dockerfile

```diff
@@ -4,14 +4,11 @@ ARG package=arcaflow_plugin_pcp
 # STAGE 1 -- Build module dependencies and run tests
 # The 'poetry' and 'coverage' modules are installed and verson-controlled in the
 # quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase image to limit drift
-FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.3.1@sha256:9767207e2de6597c4d6bd2345d137ac03661326734d4e6824840d270d3415e12 as build
+FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:stream9 as build
 ARG package
-RUN dnf -y install pcp pcp-export-pcp2json pcp-system-tools
+RUN dnf -y install pcp pcp-export-pcp2json pcp-system-tools procps-ng util-linux-core; useradd -U pcp
```

Comment:
Use `&&` instead of `;`?

Actually, since this a just a "build layer" which isn't included in the final container, just do them as separate `RUN`'s -- it's much clearer and easier to read.